torch/export/exported_program.py

```diff
@@ -566,11 +566,82 @@ def _get_updated_graph_signature(
         return transformed_ep
 
     def _check_input_constraints(self, *args):
-        from torch._export.utils import _check_input_constraints_for_graph
+        def check(cond, msg):
+            if not cond:
+                # TODO(avik): maybe add more context, e.g., graph signature
+                raise RuntimeError(msg)
+
+        from torch._export.passes.add_runtime_assertions_for_constraints_pass import (
+            _convert_range_to_int,
+        )
 
-        _check_input_constraints_for_graph(
-            self.graph, self.range_constraints, self.equality_constraints
```

Comment:
I thought we were getting rid of `equality_constraints` as well? Should be a separate PR because it might affect executorch.

Reply:
yeah, i can get rid of equality_constraints in a separate PR.